plugins/package-managers/bazel/src/test/kotlin/BazelTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.bazel
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+import org.ossreviewtoolkit.utils.test.getAssetFile
+
+class BazelTest : WordSpec({
```

Comment:
Does this unit test cover things which are not already covered by the functional test?

Reply:
Maybe not any more. It was useful during development and would probably be useful again if the functions it tests need to be updated. Should I remove it? If yes, I assume the functions it tests should be made `private` as well.

Reply:
In my view, with the current state of the tests to me they seem to be redundant and covered sufficiently by the fun test. However, if you or others really prefer having these, I'd be ok as well.

Reply:
I don't have a strong preference about keeping it or not, but I did think of one thing now that the functional tests don't cover: the function `transformBazelVersion`. However I'm not sure how useful that is. Especially since I just noticed that those tests are not even run as part of the CI. Did I fail to add them in some Gradle config file or is that expected behaviour?

Reply:
Agree, for `transformBazelVersion` I believe the test actually adds value, it clarifies what the function does.
So, only removing `parseModuleGraph()` would be the option IMO.

Reply:
Should I move the functions `parseModuleGraph` and `parseModuleGraphNode` into the class then? That would get rid of the parameter for setting `Identifier.type`.

Reply:
Makes sense to me.

Reply:
I removed `parseModuleGraph` and moved the code into `getDependencyGraph`.